internal/server/handlers/rbac/kubernetes_test.go

```diff
@@ -57,6 +57,7 @@ func TestRBAC_Kubernetes(t *testing.T) {
 					{"bob", "database-engines", "*", "*/*"},
 					{"bob", "namespaces", "*", "*"},
 					{"bob", "backup-storages", "*", "*/*"},
+					{"bob", "pod-scheduling-policies", "*", "*/*"},
```

Comment:
Same way we reference `namespaces` (2 lines above)
```suggestion
					{"bob", "pod-scheduling-policies", "*", "*"},
```

Reply:
Done